common/src/main/java/com/linecorp/line/auth/fido/fido2/common/mdsv3/metadata/BiometricAccuracyDescriptor.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2018 LINE Corporation. All rights reserved.
+ * LINE Corporation PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
+ */
+
+package com.linecorp.line.auth.fido.fido2.common.mdsv3.metadata;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class BiometricAccuracyDescriptor {
+    private Double FAR;
+    private Double FRR;
+    private Double EER;
+    private Double FAAR;
```

Comment:
The metadata v3 description is different as there is only `selfAttestedFRR` and `selfAttestedFAR` present in this structure: https://fidoalliance.org/specs/mds/fido-metadata-statement-v3.0-ps-20210518.html#authenticator-attestation-guid-aaguid-typedef

Reply:
Fixed in 17a944d.